plugins/module_utils/data_set.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2020 - 2023
+# Copyright (c) IBM Corporation 2020, 2024
```

Comment:
I'd rather keep this as a range, not a roadblock.

Reply:
per new IBM standards, it seems like the preferred format for copyright years is now `year, year`. I've DM'ed you a link to the doc in your IBM SLACK.

Reply:
Done